slime/rollout/rm_hub/ifbench_utils/instructions.py

```diff
@@ -0,0 +1,2347 @@
+# Copyright 2025 Allen Institute for AI.
```

Comment:
is it possible to pip install ifbench and import from that? the 2k loc seems terrifying...